windows/9.0.x/sitecore-xm-solr/Dockerfile

```diff
@@ -9,9 +9,9 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 ARG SOLR_VERSION
 ARG CORE_NAMES
 
-RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
-    Invoke-WebRequest -Uri ('https://archive.apache.org/dist/lucene/solr/{0}/solr-{0}.zip' -f $env:SOLR_VERSION) -OutFile 'C:\\solr.zip'; `
-    Expand-Archive -Path 'C:\\solr.zip' -DestinationPath 'C:\\temp'; `
+RUN New-Item -Path 'C:\\downloads' -ItemType 'Directory' -Force | Out-Null; `
```

Comment:
Is the TLS entry still not an issue? I thought that on some machines this entry was required to ensure the correct version of Tls was used.

Reply:
ServicePointManager is only for .NET / PowerShell, curl.exe is using "something" else.

Reply:
By default, curl.exe uses the highest level TLS available [https://ec.haxx.se/usingcurl-tls.html](https://ec.haxx.se/usingcurl-tls.html)